server/api.go

```diff
@@ -45,6 +45,7 @@ func NewAPI(p *Plugin, store store.Store) *API {
 	router.HandleFunc("/connect", api.connect).Methods("GET", "OPTIONS")
 	router.HandleFunc("/disconnect", api.handleAuthRequired(api.disconnect)).Methods("GET", "OPTIONS")
 	router.HandleFunc("/get-connected-channels", api.handleAuthRequired(api.getConnectedChannels)).Methods("GET", "OPTIONS")
```

Comment:
i dont think there is any need of adding get in the API path its method is enough for us to know that is a GET api